scripts/changelog-experimental-fix.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * For the developer preview release of CDKv2, we are stripping experimental modules from the packaged release.
+ * However, the entries from the CHANGELOG are still included, as `standard-release` has no knowledge of this process.
+ * This script -- run as part of the 'bump' process -- attempts to identify and remove CHANGELOG lines that are
+ * associated with experimental modules.
+ */
+
+const fs = require('fs-extra');
+const path = require('path');
+const { Project } = require("@lerna/project");
+
+function rewriteChangelog(changelogPath) {
+  const changelog = fs.readFileSync(changelogPath, { encoding: 'utf-8' }).split('\n');
+
+  // Find the new section of the CHANGELOG, represented by everything up to the 2nd version header.
+  // (The first version header is the new one.)
+  // This is the section we will operate on, and leave the rest untouched.
+  let searchIndex = -1;
+  const findNextVersionHeader = (str, index) => str.startsWith('## ') && index > searchIndex;
+  searchIndex = changelog.findIndex(findNextVersionHeader);
+  const secondVersionHeaderIndex = changelog.findIndex(findNextVersionHeader);
+  let newChangelogSection = changelog.splice(0, secondVersionHeaderIndex);
+
+  // Rewrite the newly-generated section of the changelog.
+  newChangelogSection = stripExperimentalPackageChangelogs(newChangelogSection);
+  newChangelogSection = stripEmptySectionHeaders(newChangelogSection);
+  newChangelogSection = stripDuplicateEmptyLines(newChangelogSection);
+  newChangelogSection = replaceBreakingChangesLine(newChangelogSection);
+  const rewrittenChangelog = [...newChangelogSection, ...changelog].join('\n');
+
+  fs.writeFileSync(changelogPath, rewrittenChangelog, { encoding: 'utf-8' });
+}
+
+function stripExperimentalPackageChangelogs(changelog) {
+  const packageNames = experimentalPackageNames();
+  const packageNamesWithDupes = [...packageNames, ...packageNames.map(pkg => pkg.replace('aws-', ''))];
```

Comment:
It's not consistent whether or not the `aws-` prefix will be there or not. Better check both I think...

Reply:
That's exactly what this does. :)  It combines the "raw" short names (with aws-) with the aws-less names, and then checks against that combined list.

Reply:
Je suis un idiot